tools/deployment/osqueryctl

```diff
@@ -69,7 +72,8 @@ exec_with_env() {
 start() {
   check_config
   if [ $OS = "darwin" ]; then
-    launchctl start $PLIST_PATH
+    cp $PLIST_INSTALLATION_PATH $PLIST_PATH
+    launchctl load $PLIST_PATH
```

Comment:
In https://github.com/facebook/osquery/pull/684: @marpaia mentioned that homebrew recommends copying instead of symlinking. 

I think one reason for this is that `launchctl` expects the label in the plist to be the same when you load and unload the plist. If you were to change the label while it was loaded, then try to unload the plist launchctl will give the error: "Could not find specified service"